features/staging_commands.feature

```diff
@@ -542,3 +542,35 @@ Feature: Enable command behaviour when attached to an UA staging subscription
            | release | fips-apt-source                                                        |
            | xenial  | https://esm.staging.ubuntu.com/fips-updates/ubuntu xenial-updates/main |
            | bionic  | https://esm.staging.ubuntu.com/fips-updates/ubuntu bionic-updates/main |
+
+
+    @series.lts
+    @uses.config.machine_type.lxd.vm
+    Scenario Outline: Commands when cloud init didn't run properly
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I delete the file `/run/cloud-init/instance-data.json`
+        And I attach `contract_token_staging` with sudo
+        Then stdout matches regexp:
+        """
+        Warning: cloud-init is present but did not run. Assuming not on cloud.
+        If you are on a cloud, you may experience unreliable behavior.
+        """
+        When I run `ua status` with sudo
+        Then stdout matches regexp:
+        """
+        Warning: cloud-init is present but did not run. Assuming not on cloud.
+        If you are on a cloud, you may experience unreliable behavior.
+        """
+        When I attempt to run `ua enable fips` with sudo
```

Comment:
Why are we using attempt here ? I think we should make sure that we still enable fips even if we detect that issue.

Reply:
Running `ua enable fips --assume-yes` skips the pre-enablement messages, since there is no confirmation step. Running it without the `--assume-yes` flag leads to an error in the test, so I am attempting here to see the message.
I tried running it with `and stdin "y\n" for the command to complete, but I had no success.

Reply:
I think we should run that verification passing no first and then enabling fips to see if everything works as expected